src/scaladoc/scala/tools/nsc/doc/model/MemberLookup.scala

```diff
@@ -100,9 +101,11 @@ trait MemberLookup extends base.MemberLookupBase {
   private def isJDK(sym: Symbol) =
     sym.associatedFile.underlyingSource.map(f => isChildOf(f, (sys.props("java.home")))).getOrElse(false)
 
+  private val sepRegex = Pattern.quote(java.io.File.separator)
+
   def jdkUrl(path: String): String = {
     if (path.endsWith(".jmod")) {
-      val tokens = path.split("/")
+      val tokens = path.split(sepRegex)
```

Comment:
```suggestion
      val tokens = path.split(File.separatorChar)
```

which uses `StringOps`, escaping the character.

Reply:
ah, even better than my suggestion